src/pytorch_lightning/trainer/trainer.py

```diff
@@ -286,9 +286,9 @@ def __init__(
                 Default: ``1.0``.
 
             logger: Logger (or iterable collection of loggers) for experiment tracking. A ``True`` value uses
-                the default ``TensorBoardLogger``. ``False`` will disable logging. If multiple loggers are
-                provided, local files (checkpoints, profiler traces, etc.) are saved in the ``log_dir`` of
-                the first logger.
+                the default ``TensorBoardLogger`` if it is installed otherwise ``CSVLogger``.
```

Comment:
```suggestion
                the default ``TensorBoardLogger`` if `tensorboard` or `tensorboardX` are installed otherwise ``CSVLogger``.
```

Reply:
If we go with `tensorboard` in the extras then we remove the reference to `tensorboardX` here